docs/source/features/activation_recomputation.rst

```diff
@@ -0,0 +1,33 @@
+Activation Recomputation
+========================
+
+The input activations of network layers are stored in the device memory to compute the gradients in back-propagation.
+The input activation stores easily saturate the device memory when training a LLM with a large sequence length or a large micro-batch size.
+Check-pointing a few activations and recomputing the rest of activations is a common technique to reduce the need of device memory.
```

Comment:
Check-pointing a few activations and recomputing the rest of the activations is a common technique to reduce the need for device memory.